spec/config/lint-on-fly-spec.coffee

```diff
@@ -0,0 +1,11 @@
+describe "The Lint on the Fly Configuration Option", ->
```

Comment:
I like to keep small and similar tests in the same file.  I think the only difference between these is the `describe` / `it` text and `configString`.

Even if you combine them, I'm not opposed to splitting larger more complex tests into their own files. I can see where you might want `status-icon-position-spec` to be its own file.

As I wrote this, I don't think either approach is significantly or obviously better than the other, but those are my thoughts on it.